### PR TITLE
release 1.8.0

### PR DESCRIPTION
**Pull request by creisle**
New Features
- cnvnator support
- breakseq vcf support

Improvements
- masking, dgv_annotation, and template_metadata files are now all optional
- Moved optional dependencies from the install_requires to tests_require and extras_require to improve setup
- Additional documentation and reorganization of the user manual
- Moved the complexity filtering to the summary stage

BugFixes
- Hard clipped read error during read alignment to contig
- vcf input option error where it was not being properly added to the supported tools

### CODE REVIEW

File: mavis/tools.py [hunk 1]
```diff
@@ -202,8 +202,13 @@ def _parse_vcf_record(record):
...
     for alt in record.alts if record.alts else [None]:
         info = {}
-        for entry in record.info.items():
-            info[entry[0]] = entry[1:] if len(entry[1:]) > 1 else entry[1]
+        for key in record.info.keys():
+            try:
+                info[key] = record.info[key]
+            except UnicodeDecodeError as e:
+                # Invalid character in this info field.
+                # TODO: log a warning?
```
Comment (amirmhzadeh) [inline mavis/tools.py]: Log a warning then. Why is it still TODO?

File: README.md [hunk 2]
```diff
@@ -22,36 +24,80 @@ The general [MAVIS](http://mavis.bcgsc.ca) pipeline consists of six main stages
...
+by running without any arguments, or by giving the -h/--help option
+
+    mavis -h
+
+Help sub-menus can be found by giving the pipeline step followed by no arguments or the -h options
+
+    mavis cluster -h
+
+Common problems and questions are addressed on the [wiki](https://github.com/bcgsc/mavis/wiki/Help-and-Frequently-Asked-Questions).
+If you have a question or issue that is not answered there (or already an github issue) please submit
```
Comment (amirmhzadeh) [inline README.md]: already "a" github issue

File: setup.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import re
 
 
-VERSION = '1.7.0'
+VERSION = '1.7.1'
```
Comment (amirmhzadeh) [inline setup.py]: Is this version correct?

Comment (amirmhzadeh) [reply setup.py]: nvm

File: mavis/cluster/constants.py [hunk 4]
```diff
@@ -29,5 +29,5 @@
     'out prior to clustering')
 DEFAULTS.add(
     'limit_to_chr', ChrListString(';'.join([str(x) for x in range(1, 23)] + ['X', 'Y'])), cast_type=ChrListString,
```
[the remark this answers is not in the document]
Comment (creisle) [reply mavis/cluster/constants.py]: This is just the default. To add the mitochondrial chromosome you can just change the config file

**Pull request merged**